api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -387,7 +388,7 @@
       </fields>
       <codeSegments>
         <codeSegment>
-          <version>4.0.0/4.1.0</version>
+          <version>4.0.0/4.0.99</version>
```

Comment:
@gnodet What is this supposed to mean?

Reply:
This is closed ranged version notation.  That's defined by modello...

Reply:
Ah, sorry. I get it now. I thought this element was for documentation purposes

Reply:
Sadly, modello "version range" predates (AFAIK, someone may correct me) Maven ranges, and has totally different  syntax...